### DISCUSSION TITLE
Add "10 Key" mod back

### DISCUSSION THREAD

**Discussion by cuhoangdk** [ppy/osu#18261] [Ideas] [answered]
there're many 10K map ranked now so can we just add this mod back to practice with convert?

**Comment by bdach** [comment, marked as answer]
what do you mean by "add back"? it never went anywhere?

![image](https://user-images.githubusercontent.com/20418176/168440023-2ecf0bc0-685f-4b90-b62b-47ce225cceba.png)

**Reply by cuhoangdk** [reply]
I mean it wasn't on stable

**Reply by bdach** [reply]
stable is feature locked.

**Reply by cuhoangdk** [reply]
D:
